docs-web/src/main/java/com/sismics/docs/rest/resource/ScoreResource.java

```diff
@@ -97,4 +101,61 @@ public Response add(
                 // .add("total_score", score.getTotalScore());
         return Response.ok().entity(response.build()).build();
     }
+
+    /**
+     * Get all scores on a document.
+     *
+     * @api {get} /scores/:id Get scores
+     * @apiName GetScore
+     * @apiGroup Scores
+     * @apiParam {String} id Document ID
+     * @apiParam {String} share Share ID
+     * @apiSuccess {Object[]} scores List of scores
+     * @apiSuccess {String} scores.id Score ID
+     * @apiSuccess {String} scores.documentId DocumentId
+     * @apiSuccess {Integer} scores.skills skills
+     * @apiSuccess {Integer} scores.experience experience
+     * @apiSuccess {Integer} scores.transcriptGPA transciptGPA
+     * @apiSuccess {Integer} scores.match match
+     * @apiSuccess {Number} scores.create_date Create date (timestamp)
+     * @apiError (client) NotFound Document not found
+     * @apiPermission none
+     * @apiVersion 1.5.0
+     *
+     * @param documentId DocumentID
+     * @return Response
+     */
+    @GET
+    @Path("{documentId: [a-z0-9\\-]+}")
+    public Response get(@PathParam("documentId") String documentId,
+            @QueryParam("share") String shareId) {
```

Comment:
Could you briefly explain what the "share" query param is used for and where the shareId will be retrieved from?

Reply:
Not sure, probably an already bound environment variable, and only used for checking permission. Refer to CommentResource.java for its usage.